docker/docker-compose.yml

```diff
@@ -212,17 +213,22 @@ services:
         condition: service_healthy # this waits until healthcheck passes
       aws_cli:
         condition: service_started # this doesn't
+      casework:
+        condition: service_healthy
+      documents:
+        condition: service_healthy
+      info:
+        condition: service_healthy
```

Comment:
Does workflow really depend on all these? (Or rather, do we really need to wait for all of these to be up before we can bring workflow up?)

Reply:
workflow is a client of all those, workflow is flaky on start up. its less flaky with these.
It could do with more investigation.